### DISCUSSION TITLE
World Switcher Plugin potentially synesthesia friendly?

### DISCUSSION THREAD

**Discussion by DolphinsJ** [runelite/runelite#15420] [Ideas]
Due to having synesthesia (on top of dyslexia/dyscalcula) my brain doesn't process the text in games very well, it can be hard for me to play certain ones. In fact doing any world hopping in runescape tends to look like this:
![unknown](https://user-images.githubusercontent.com/110275452/181876076-75999e1a-75d9-47bf-b05b-24ea2c2d3477.png)
Because I see the plugin like this:
![quick_hop](https://user-images.githubusercontent.com/110275452/181876079-e4565aee-8f6f-4113-ae9d-9c87a53cace4.png)
It's harder for me to world hop because I have to cross reference my worlds between both methods of world hopping, it'd be very helpful just to have a members or free icon in the world switcher plugin so I can just use one. 

Thank you for considering.

**Comment by Enriath** [comment]
If you never want to be able to switch to a F2P world, it's possible to disable them in the World Hopper config with `Show subscription types`.
